lib/pages/mysite-page.js

```diff
@@ -0,0 +1,29 @@
+// This is the page shown after clicking on the site name from side bar
+
+import { By as by, until } from 'selenium-webdriver';
+import config from 'config';
+
+import BaseContainer from '../base-container.js';
+import * as driverHelper from '../driver-helper.js';
+
+export default class MySitePage extends BaseContainer {
+	constructor( driver, visit = false ) {
+		let url = config.get( 'calypsoBaseURL' );
+		if ( config.has( 'liveBranch' ) && config.get( 'liveBranch' ) === 'true' ) {
+			url = url + '?branch=' + config.get( 'branchName' );
+		}
+		super( driver, by.css( '.web-preview__frame' ), visit, url );
+	}
+	isWebPreviewPresent() {
```

Comment:
I'm not sure this is strong enough of an assertion. This will show whether there is a frame to hold content, but it could be empty.

If you look at post-preview-component you'll see where it switches to the iFrame and asserts some content is shown, for a site we may just want to assert that `body.home` is present in the frame or something quite high level.